examples/generate.sh

```diff
@@ -144,24 +145,34 @@ if [[ "$DOMAIN_NAME" = "null" ]]; then
 fi
 CACERT_ORIGINAL=$(echo "$OPENSTACK_CLOUD_CONFIG_PLAIN" | yq r - clouds.${CLOUD}.cacert)
 
+# use only the selected cloud not the whole clouds.yaml
+OPENSTACK_CLOUD_CONFIG_SELECTED_CLOUD=$(echo "${OPENSTACK_CLOUD_CONFIG_PLAIN}" | yq r - clouds.${CLOUD} | yq p - clouds.${CLOUD})
+
 # Basic cloud.conf, no LB configuration as that data is not known yet.
-export OPENSTACK_CLOUD_PROVIDER_CONF="[Global]
-          auth-url=$AUTH_URL
-          username=\"$USERNAME\"
-          password=\"$PASSWORD\"
-          tenant-id=\"$PROJECT_ID\"
-          domain-name=\"$DOMAIN_NAME\"
+OPENSTACK_CLOUD_PROVIDER_CONF="
+[Global]
+auth-url=$AUTH_URL
+username=\"$USERNAME\"
+password=\"$PASSWORD\"
+tenant-id=\"$PROJECT_ID\"
+domain-name=\"$DOMAIN_NAME\"
 "
 if [[ "$CACERT_ORIGINAL" != "null" ]]; then
   OPENSTACK_CLOUD_PROVIDER_CONF="$OPENSTACK_CLOUD_PROVIDER_CONF
-          ca-file=\"${CACERT_ORIGINAL}\"
+ca-file=\"${CACERT_ORIGINAL}\"
   "
 fi
 if [[ "$REGION" != "null" ]]; then
   OPENSTACK_CLOUD_PROVIDER_CONF="$OPENSTACK_CLOUD_PROVIDER_CONF
-          region=\"${REGION}\"
+region=\"${REGION}\"
   "
 fi
+
+export OPENSTACK_CLOUD_PROVIDER_CONF6
+OPENSTACK_CLOUD_PROVIDER_CONF6=$(echo "${OPENSTACK_CLOUD_PROVIDER_CONF}" | sed -e 's/^/      /')
+export OPENSTACK_CLOUD_PROVIDER_CONF10
+OPENSTACK_CLOUD_PROVIDER_CONF10=$(echo "${OPENSTACK_CLOUD_PROVIDER_CONF}" | sed -e 's/^/          /')
```

Comment:
I didn't get this CONF6/CONF10 ... what's the meaning of this?

Reply:
The problem is that I sometimes need the config with indent 6 (KubeadmConfig) and sometimes with indent 10 (KubeadmConfigTemplate). Seems to be the easiest way to just have to vars for envsubst.

Reply:
ok, got it thanks